dev_guide/routes.adoc

```diff
@@ -60,7 +61,35 @@ spec:
     name: frontend
 ----
 ====
+endif::[]
+ifdef::openshift-online[]
+Using the CLI, the following example creates an unsecured route:
+
+----
+$ oc expose svc/frontend
+----
 
+The new route inherits the name from the service unless you specify one
+using the `--name` option.
+
+.YAML Definition of the Unsecured Route Created Above
+====
+[source,yaml]
+----
+apiVersion: v1
+kind: Route
+metadata:
+  name: frontend
+spec:
+  host: frontend-projectname.{product-title} {product-version} hostname
```

Comment:
What's up with `.{product-title} {product-version} hostname`? Anyway, `host` should not be specified at all.